helm/coder/values.yaml

```diff
@@ -113,6 +113,8 @@ coder:
     annotations: {}
     # coder.serviceAccount.name -- The service account name
     name: coder
+    # coder.serviceAccount.name -- Whether to create the service account or use existing service account
+    create: true
```

Comment:
I think defaulting to false is better in this scenario. So maybe "disableCreate" or something?

Curious what @ericpaulsen thinks

Reply:
i agree - defaulting to `false` is best.